src/API/Google/Settings.php

```diff
@@ -387,6 +387,58 @@ public function get_store_address(): AccountAddress {
 		return $mc_address;
 	}
 
+	/**
+	 * Check whether the address has errors
+	 *
+	 * @param AccountAddress $address to be validated.
+	 *
+	 * @return array
+	 */
+	public function has_address_errors( AccountAddress $address ): array {
+		if ( ! $address instanceof AccountAddress ) {
+			return [];
+		}
+
+		/** @var WC $wc */
+		$wc = $this->container->get( WC::class );
+
+		$country   = $address->getCountry();
+		$countries = $wc->get_wc_countries();
+
+		$locale          = $countries->get_country_locale();
+		$locale_settings = isset( $locale[ $country ] ) ? $locale[ $country ] : [];
```

Comment:
💅  WDYT of `$locale[ $address->getCountry() ] ?? []` 

Info --> https://www.php.net/manual/en/migration70.new-features.php#:~:text=Null%20coalescing%20operator%20%C2%B6

Reply:
I think both versions are given the same result in this context. In this case, I think the ternary operator would give better readability but is adding more verbosity to the code.

Just to keep the same consistency with the other changes I have updated the null coalescing operator. See [#7b1c89b](https://github.com/woocommerce/google-listings-and-ads/pull/1169/commits/7b1c89bf566dfd9c097f84ac6e9d256a54f7b584)